model/src/gateway/payload/member_update.rs

```diff
@@ -12,7 +12,13 @@ pub struct MemberUpdate {
     /// Whether the user has yet to pass the guild's [Membership Screening]
     /// requirements.
     ///
+    /// Note: This field is still under refactoring by discord. For more info,
```

Comment:
```suggestion
    /// Note: This field is still under refactoring by Discord. For more info,
```

Not sure if it should say "Note", but Discord needs to be capitalized to match our other docs